orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/target/HostList.java

```diff
@@ -19,4 +19,11 @@ public String getTarget() {
     public String getType() {
         return "list";
     }
+
+    @Override
+    public String toString() {
+        return "HostList{" +
+                "targets=" + targets +
```

Comment:
Please join the name of the targets as the logged value could be only a reference based on the implementation of the Collection.

Reply:
fixed, please check again